bokeh/client/session.py

```diff
@@ -153,11 +153,16 @@ def __init__(self, session_id=DEFAULT_SESSION_ID, io_loop=None, url=DEFAULT_SERV
         self._connection = ClientConnection(session=self, io_loop=io_loop, url=url)
 
         self._current_patch = None
+        self._callbacks = []
 
     def _attach_document(self, document):
         self._document = document
         self._document.on_change(self._document_changed)
 
+    def add_periodic_callback(self, cb, callback_time):
```

Comment:
And... we probably want to trigger:

`CallbackAdded` here https://github.com/bokeh/bokeh/blob/master/bokeh/document.py#L182

and 

`CallbackRemoved` here https://github.com/bokeh/bokeh/blob/master/bokeh/document.py#L191